test/twine_test.coffee

```diff
@@ -277,10 +277,10 @@ suite "Twine", ->
       node = setupView(testView, context = fn: @spy())
       data = {test: 'bla123'}
 
-      $(node).trigger 'submit', data
+      triggerEvent node, 'submit', data
 
       assert.isTrue context.fn.calledOnce
-      assert.isTrue context.fn.calledWith(data)
+      assert.isTrue context.fn.calledWith(data), 'fn called with data'
```

Comment:
https://github.com/Shopify/twine/pull/47/files#diff-54cce0c8766de7686ed6f38e8fe0c71dR528 is another good place to debugger